products/opensuse/main.pm

```diff
@@ -38,6 +38,10 @@ sub is_leap {
     return get_var('VERSION') =~ /^42:S/;
 }
 
+sub is_leap_42_3_or_newer {
+    return is_leap && check_var('VERSION', '42.3');
+}
+
```

Comment:
Also `main_common.pm` might be a better home. I'd have a use-case for the function for conditional load of `snapper_thin_lvm` test in it.